### DISCUSSION TITLE
Activity not showing up in profile activity tracker

### DISCUSSION THREAD

**Discussion by derekjtong** [community/community#134003] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Hello! I wanted to do at least one contribution a day. Not company mandated or anything, I'm just a little obsessed. Last week I did a PR review, but it seems like it's not appearing in the activity tracker. What gives?

![Screenshot 2024-07-29 at 8 00 00 PM](https://github.com/user-attachments/assets/4a7a1618-e3ee-4dcd-9417-fc520d5fecf0)

**Comment by std-enigma** [comment]
There are two possible reasons:

- You just have to give it a little bit more time to update
- You review contribution didn't met the requirements to update the graph _for more information_: [see](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/why-are-my-contributions-not-showing-up-on-my-profile)

**Comment by std-enigma** [comment]
I hope this answers your discussion

**Comment by derekjtong** [comment]
Thank you. I'm still a little confused because it showed up on Friday, but then disappeared on Monday. Perhaps it was because I requested changes on Friday, then on Monday I approved it?
